src/App.tsx

```diff
@@ -15,71 +15,79 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component <{}, State> {
+  state = {
+    selectedGood: goods[goods.indexOf('Jam')] || goods[0],
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  handler = (good:string) => {
```

Comment:
Consider renaming the 'handler' function to a more descriptive name, such as 'handleSelectGood'.